tests/test_gzip.py

```diff
@@ -0,0 +1,13 @@
+from bocadillo import API
+
+
+def test_if_gzip_enabled_and_response_is_compressed():
```

Comment:
I think you meant 'then' here:

```suggestion
def test_if_gzip_enabled_then_response_is_compressed():
```